web/src/lib/components/faces-page/show-hide.svelte

```diff
@@ -1,24 +1,46 @@
+<script lang="ts" context="module">
+  export enum ToggleVisibilty {
+    HIDE_ALL = 'hide-all',
+    HIDE_UNNANEMD = 'hide-unnamed',
+    VIEW_ALL = 'view-all',
+  }
+
+  export const changeVisibility = (toggleVisibility: ToggleVisibilty) => {
+    if (toggleVisibility === ToggleVisibilty.VIEW_ALL) {
+      return ToggleVisibilty.HIDE_UNNANEMD;
+    } else if (toggleVisibility === ToggleVisibilty.HIDE_UNNANEMD) {
+      return ToggleVisibilty.HIDE_ALL;
+    } else {
+      return ToggleVisibilty.VIEW_ALL;
+    }
+  };
+</script>
+
 <script lang="ts">
   import { fly } from 'svelte/transition';
   import CircleIconButton from '../elements/buttons/circle-icon-button.svelte';
   import { quintOut } from 'svelte/easing';
-  import { createEventDispatcher } from 'svelte';
   import LoadingSpinner from '$lib/components/shared-components/loading-spinner.svelte';
-  import { mdiClose, mdiEye, mdiEyeOff, mdiRestart } from '@mdi/js';
+  import { mdiClose, mdiEye, mdiEyeOff, mdiEyeSettings, mdiRestart } from '@mdi/js';
   import { locale } from '$lib/stores/preferences.store';
   import Button from '$lib/components/elements/buttons/button.svelte';
 
-  const dispatch = createEventDispatcher<{
-    close: void;
-    reset: void;
-    change: void;
-    done: void;
-  }>();
-
   export let showLoadingSpinner: boolean;
-  export let toggleVisibility: boolean;
+  export let toggleVisibility: ToggleVisibilty = ToggleVisibilty.VIEW_ALL;
   export let screenHeight: number;
   export let countTotalPeople: number;
+  export let onClose = () => {};
+  export let onReset = () => {};
+  export let onChange = () => {};
+  export let onDone = () => {};
```

Comment:
```suggestion
  export let onClose: () => void;
  export let onReset: () => void;
  export let onChange: () => void;
  export let onDone: () => void;
```
This is what you want, right?

Reply:
We use both in the codebase, what should we use ?

Reply:
I would personally prefer this, as it better aligns with other exports, e.g.`export let assetId: string`